### DISCUSSION TITLE
Verison 3.1.0 is now available.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1265] [Announcements]
We are happy to announce the availability of version 3.1.0.

This is a very important release to the active maintainers, because we have found a new home: https://github.com/pymodbus-dev/pymodbus which is 100% FOSS. 

The move from riptideio (a company) was made possible thanks work done by @dhoomakethu 

We have a lot of work ahead of us, pymodbus is already known in the FOSS community as THE python modbus library, but it needs maintenance to keep staying efficient. Release 3.0 brought us a long way, now it is time to focus on efficiency as well as test coverage. Version 3.2 is scheduled to contain major efficiency improvements in the client as well as a brand new web based simulator.

If you use pymodbus, please think about:
- At least share your usage, as an example for others (add to examples/contrib), make a pull request and see how fast we respond.
- Having troubles, then open a discussion or an issue. We will guide you with the information we need.
- Found a bug and solved it, BIG hurray, you can save others having trouble by submitting a Pull Request.
- You are not a developer but using pymodbus, let us know your experience...we are often too much developers and forget real problems.
- If you do not develop, then you might help us with documentation or web design (the new simulator).
- if nothing else, a short "I use pymodbus and it works" helps keep the maintainers motivated (but of course it is just a suggestion!!!!) 😃 

We look forward to hear from you....we work hard to make your day just a bit better, so please let us hear from you (positive or negative).

**Comment by MAKOMO** [comment]
Thank you for your wonderful work! We see the many hours you spent by following your commits and PRs. Amazing!

We use pymodbus in [Artisan](https://github.com/artisan-roaster-scope/artisan), an open-source software for coffee roast logging. We started in 2013 to add MODBUS support to access temperature PID devices as from Fuji and others via serial MODBUS RTU, by then using the minimalmodbus lib. In summer 2014 we moved over to pymodbus v1.3.0 from GoogleCode, allowing us to adding MODBUS TCP/UDP support to Artisan and have every since relied on it.

Your lib enabled the roasting of wonderful coffee, by home-roasters and many small artisan roasting companies connecting Artisan to various types of roasting machines in virtual any country on this planet. All of this would not have been possible without such a reliable open-source MODBUS library.

Keep it up!

**Comment by alexrudd2** [comment]
I work for [NuMat Technologies](https://github.com/numat/), and we use a bunch of laboratory and industrial equipment that speaks MODBUS.  We open-source our drivers for several types of equipment:
https://github.com/numat/watlow
https://github.com/numat/clickplc
https://github.com/numat/productivity
https://github.com/numat/midas

My day job is engineering, but sometimes I get some time to maintain those drivers.  Slowly learning `pymodbus` as well :)
